pymc_experimental/tests/distributions/test_discrete_markov_chain.py

```diff
@@ -0,0 +1,91 @@
+import numpy as np
+import pymc as pm
+
+# general imports
```

Comment:
Can you add a test for the random and moment functions?

Reply:
The moment function was removed, but tests were added for random draws and change_dist_size.